py/desispec/night_qa.py

```diff
@@ -796,26 +825,27 @@ def create_petalnz_pdf(outpdf, night, prod, survey="main", dchi2_threshold=25):
                 log.warning("{} : no file".format(fn))
             else:
                 istileid = True
-                d = Table(
-                    fitsio.read(
-                        fn,
-                        ext="ZMTL",
-                        columns=[
-                            "TARGETID",
-                            "{}DESI_TARGET".format(prefix_key), "{}BGS_TARGET".format(prefix_key),
-                            "Z", "ZWARN", "SPECTYPE", "DELTACHI2",
-                            "Z_QN", "Z_QN_CONF", "IS_QSO_QN",
-                        ],
-                    )
-                )
+                d = Table.read(fn, hdu="ZMTL")
+                # AR rename *DESI_TARGET and *BGS_TARGET to DESI_TARGET and BGS_TARGET
+                keys, _, _ = main_cmx_or_sv(d)
+                d.rename_column(keys[0], "DESI_TARGET")
+                d.rename_column(keys[1], "BGS_TARGET")
```

Comment:
nice trick to simplify downstream code.

Reply:
thanks.
I know there is some fragility here: it only works because Adam had the nice idea to reserve the same bit values for LRG/ELG/QSO in the *desi_mask and BGS_BRIGHT/BGS_FAINT in the *bgs_mask for sv1/sv2/sv3/main.
trying to display other tracers is likely to break that; but I don't foresee that for that plot, so I've picked this shortcut approach.